maven-plugin-plugin/pom.xml

```diff
@@ -296,7 +276,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-plugin-plugin</artifactId>
-        <version>3.6.0</version><!-- cannot use ${mavenPluginToolsVersion} property because release plugin would try to update -->
```

Comment:
is it not valid?
will work during release?

Reply:
Um, no idea, I tend to avoid m-release-p, never used it (aside of Maven projects). But, if this (otherwise perfectly valid) construct "does not work" with m-release-p, isn't the release plugin that needs to be fixed? Or we want to sprinkle "workarounds" all around as "release plugin would try to update"?

Reply:
This is the origin of the comment: https://github.com/apache/maven-plugin-tools/commit/3b54fdbc289fbc33db2366c37579e13a0e087a08  the commit message to me looks like an issue description :smile: (and a very old one)

Reply:
https://issues.apache.org/jira/browse/MRELEASE-799
maybe it is a bug only for artifact dependency not for plugins versions ...

according to issue should be reproducible with
```
mvn release:prepare -DdryRun=true ...
```

Reply:
My take on this https://github.com/apache/maven-release/pull/100